package.json

```diff
@@ -20,6 +20,7 @@
     "expo": "~40.0.0",
     "expo-font": "~8.4.0",
     "expo-status-bar": "~1.0.3",
+    "firebase": "7.9.0",
```

Comment:
If you are on firebase 9 please checkout master for a more up to date reference:
https://github.com/mobinni/MealsToGo/blob/master/package.json